tests/models/test_ballot_choices.py

```diff
@@ -0,0 +1,72 @@
+import pytest
```

Comment:
Your tests seem quite rigorous, so I'm just going to skim then and not comment. I'll run them, of course!